public/sandbox-data/apprenticeship-levy/employers/198QEVYVMUYLQ/employed/AA789012C.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": 200,
+  "jsonBody": {
+    "empref": "198/QEVYVMUYLQ",
+    "nino": "AA789012C",
```

Comment:
Use of Nino

Reply:
Ill address this in a later ticket. This is a bigger piece of work as App-levy is using itself as a stub. We need to create a separate microservice for this.